CHANGELOG.adoc

```diff
@@ -17,6 +17,12 @@
 |===
 | | Description | PR
 
+
+| ✨
+| Adding base64 data handling to ping update command
+| https://github.com/knative/client/pull/1392[#1392]
+| https://github.com/knative/client/pull/1388[#1388]
```

Comment:
This should be rather one line, otherwise it breaks the table rendering.

E.g.:
```suggestion
| https://github.com/knative/client/pull/1388[#1388], https://github.com/knative/client/pull/1392[#1392]
```